onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2313,10 +2313,18 @@ def aten_fix(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_flip(self: TensorType, dims: Sequence[int]) -> TensorType:
+@torch_op("aten::flip")
+def aten_flip(self: TTensor, dims: Sequence[int]) -> TTensor:
     """flip(Tensor self, int[] dims) -> Tensor"""
 
-    raise NotImplementedError()
+    dims_tensor = op.Constant(value_ints=dims)
```

Comment:
This should fail as Constant requires a compile time constant.

Reply:
So what should we do for this scenario?

Reply:
Kind of like this: https://github.com/microsoft/onnx-script/blob/1e8d764a9be04323d7171e4d5f511332790cb809/onnxscript/function_libs/torch_aten/ops/core.py#L5503-L5513

Reply:
We can use expand instead

Reply:
But here we should actually expend an INT64 tensor

Reply:
> 

Thanks @titaiwangms , but, seems the aten_sym_size is not enabled the testing, because no test case in OPS_DB.
I test it with below code:
```
import numpy as np
a = np.random.rand(2,3)
dim = 1
r=aten_sym_size(a, dim)
print(r)
exit(0)
```
It will failed at line 5511 and 5512.  The should be op.Reshape(dim, op.Constant(value_ints=[1])).

But for my case: the dim is a tuple (0,1,2), op.Reshape() or op.Expand() both cannot work.

Reply:
> We can use expand instead

op.Expand(A, shape) cannot work when A is a list.

Reply:
Can we annotate dims as INT64? This way it will be converted to a tensor

Reply:
I skip the test case when dims=(empty) because it is meaningless, torch also return error for this case.

Reply:
fixed